app/controllers/application_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ApplicationController < ActionController::API
+
+  def build_overdue_array(rentals_array)
```

Comment:
Why is this here?  This looks like it should be in the model!

Come to think of it it looks like it is!